libraries/standard/mqtt/integration-test/test_config.h

```diff
@@ -56,39 +56,40 @@
  * the instructions in https://mosquitto.org/ for running a Mosquitto broker
  * locally.
  */
-/*#define BROKER_ENDPOINT    "architag-ec2-mqtt.freeddns.org" */
-#define BROKER_ENDPOINT           "test.mosquitto.org"
+#ifndef BROKER_ENDPOINT
+    #define BROKER_ENDPOINT    "test.mosquitto.org"
+#endif
 
 /**
  * @brief Length of MQTT server host name.
  */
-#define BROKER_ENDPOINT_LENGTH    ( ( uint16_t ) ( sizeof( BROKER_ENDPOINT ) - 1 ) )
+#ifndef BROKER_ENDPOINT_LENGTH
+    #define BROKER_ENDPOINT_LENGTH    ( ( uint16_t ) ( sizeof( BROKER_ENDPOINT ) - 1 ) )
+#endif
 
 /**
  * @brief MQTT server port number.
  *
  * In general, port 8883 is for secured MQTT connections.
  */
-#define BROKER_PORT               ( 8883 )
+#define BROKER_PORT    ( 8883 )
 
 /**
  * @brief Path of the file containing the server's root CA certificate.
  *
  * This certificate should be PEM-encoded.
  */
-/*#define SERVER_CERT_PATH          "/home/ubuntu/AWS_Certificates/Amazon_Root_CA_1.txt" */
-#define SERVER_ROOT_CA_CERT_PATH    "/home/ubuntu/Repos/aws-iot-device-sdk-embedded-C/libraries/standard/mqtt/integration-test/mosquitto.org.crt"
-
-/**
- * @brief Length of path to server certificate.
- */
-#define SERVER_CERT_PATH_LENGTH     ( ( uint16_t ) ( sizeof( SERVER_CERT_PATH ) - 1 ) )
+#ifndef SERVER_ROOT_CA_CERT_PATH
+    #define SERVER_ROOT_CA_CERT_PATH    ""
```

Comment:
Perhaps we should `#error` here instead of setting an empty string? So whoever is running these tests will know exactly where to fix it.

Reply:
Update to use `#error` instead